src/lib/AppRegistry.tsx

```diff
@@ -87,6 +88,13 @@ interface ArtworkProps {
 
 const Artwork: React.SFC<ArtworkProps> = props => <ArtworkRenderer {...props} />
 
+interface PartnerProps {
+  partnerID: string
+  safeAreaInsets: SafeAreaInsets
+}
+
+const Partner: React.SFC<PartnerProps> = props => <PartnerRenderer {...props} />
```

Comment:
We should add some basic analytics tracking here before shipping. The Artwork page is the most recently-added component so it should represent the most up-to-date analytics schema; we can mimic that schema here and then get feedback from @anipetrov before shipping 👍